extensions/wikia/ArticleComments/classes/ArticleComment.class.php

```diff
@@ -24,7 +24,7 @@ class ArticleComment {
 	public $mNamespace;
 
 	/** @var array */
-	public $mMetadata;
+	private $mMetadata;
 
 	private $mText = false; // parsed HTML
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Move this trailing comment on the previous empty line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS139)